tests/Session/test_profile.py

```diff
@@ -47,6 +49,64 @@ def testListFoodStorages(self):
 
         self.profile.addFoodStorage(FRIDGE, fridge_name)
         self.assertEqual([freezer_name, fridge_name], self.profile.listFoodStorages(FOOD_STORAGE)) 
+    
+    def testSave(self):
+        TEST_PROFILES_PATH = "tests/TestUtils/TestProfiles/"
+        
+        # Profile with no food storages
+        TEST_PROFILE_NAME_1 = "leeza"
+        PROFILE_NO_FOOD_STORAGES = "profile_no_foodStorages"
+        profile1 = Profile(TEST_PROFILE_NAME_1)
+        profile1.save()
+        with open(TEST_PROFILES_PATH + PROFILE_NO_FOOD_STORAGES + JSON_EXTENSION) as user_file:
+            testProfile = json.load(user_file)
+        with open(PROFILES_PATH + SLASH + TEST_PROFILE_NAME_1 + JSON_EXTENSION) as user_profile:
+            savedProfile = json.load(user_profile)
+
+        self.assertEqual(testProfile, savedProfile)
+
+        # Profile with one food storage ("fridge")
+        TEST_PROFILE_NAME_2 = "amanda"
+        PROFILE_W_ONE_FOOD_STORAGE = "profile_fridge"
+        profile2 = Profile(TEST_PROFILE_NAME_2)
+        profile2.addFoodStorage(FRIDGE, fridge_name)
+        profile2.save()
+        with open(TEST_PROFILES_PATH + PROFILE_W_ONE_FOOD_STORAGE + JSON_EXTENSION) as user_file:
+            testProfile = json.load(user_file)
+        with open(PROFILES_PATH + SLASH + TEST_PROFILE_NAME_2 + JSON_EXTENSION) as user_profile:
+            savedProfile = json.load(user_profile)
+
+        self.assertEqual(testProfile, savedProfile)
+
+        # Profile with multiple food storages ("fridge" and "freezer")
+        TEST_PROFILE_NAME_3 = "josh"
+        PROFILE_W_MULTI_FOOD_STORAGE = "profile_fridge_freezer"
+        profile3 = Profile(TEST_PROFILE_NAME_3)
+        profile3.addFoodStorage(FRIDGE, fridge_name)
+        profile3.addFoodStorage(FREEZER, freezer_name)
+        profile3.save()
+        with open(TEST_PROFILES_PATH + PROFILE_W_MULTI_FOOD_STORAGE + JSON_EXTENSION) as user_file:
+            testProfile = json.load(user_file)
+        with open(PROFILES_PATH + SLASH + TEST_PROFILE_NAME_3 + JSON_EXTENSION) as user_profile:
+            savedProfile = json.load(user_profile)
+
+        self.assertEqual(testProfile, savedProfile)
+
+        # ## will add after merge with add_food command ##
```

Comment:
your tests here look good so far! just waiting for the last test when add_food gets merged!

Reply:
Thank you!